backend/api/Areas/Property/Controllers/ParcelController.cs

```diff
@@ -177,6 +177,26 @@ public IActionResult UpdateParcel(int id, [FromBody] Model.ParcelModel model)
             return new JsonResult(_mapper.Map<Model.ParcelModel>(parcel));
         }
 
+        /// <summary>
+        /// Update the specified parcel financials values in the datasource if the user is allowed.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="model"></param>
+        /// <returns></returns>
+        [HttpPut("{id}/financials")]
+        [HasPermission(Permissions.PropertyEdit)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(Model.ParcelModel), 200)]
```

Comment:
possibly missing response error types.

Reply:
@devinleighsmith I dont understand what you mean

Reply:
From the above file:

```
[HttpPut("{id}/financials")]
        [HasPermission(Permissions.PropertyEdit)]
        [Produces("application/json")]
        [ProducesResponseType(typeof(Model.BuildingModel), 200)]
        [ProducesResponseType(typeof(ApiModels.ErrorResponseModel), 400)]
        [ProducesResponseType(typeof(ApiModels.ErrorResponseModel), 403)]
```

Reply:
the 400 / 403 response